interface-definitions/load-balancing-inbound.xml.in

```diff
@@ -0,0 +1,220 @@
+                  <valueHelp>
+                    <format>sctp</format>
+                    <description>SCTP Protocol</description>
+                  </valueHelp>
+                  <constraint>
+                    <regex>^(tcp|udp|sctp)$</regex>
+                  </constraint>
+                  <constraintErrorMessage>The load balancing protocol must be one of tcp, udp, and sctp</constraintErrorMessage>
+                </properties>
+                <defaultValue>tcp</defaultValue>
+              </leafNode>
+              <leafNode name="virtual-ip">
+                <properties>
+                  <help>Virtual server IP</help>
+                  <valueHelp>
+                    <format>ipv4</format>
+                    <description>IPv4 VIP for the virtual server</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>ipv6</format>
+                    <description>IPv6 VIP for the virtual server</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="ip-address"/>
+                  </constraint>
+                </properties>
+              </leafNode>
+              <leafNode name="virtual-port">
+                <properties>
+                  <help>Virtual server Port</help>
+                  <valueHelp>
+                    <format>u32:</format>
+                    <description>The port number of the virtual server</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-65535"/>
+                  </constraint>
+                  <constraintErrorMessage>The port number of the virtual server must be between 0 and 65535</constraintErrorMessage>
+                </properties>
+              </leafNode>
+              <leafNode name="mode">
+                <properties>
+                  <help>Execute load balancing operation mode</help>
+                  <completionHelp>
+                    <list>dr tun nat</list>
+                  </completionHelp>
+                  <valueHelp>
+                    <format>dr</format>
+                    <description>Direct Routing mode</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>tun</format>
+                    <description>Tunnel mode(IPIP Tunnel)</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>nat</format>
+                    <description>NAT mode</description>
+                  </valueHelp>
+                  <constraint>
+                    <regex>^(dr|tun|nat)$</regex>
+                  </constraint>
+                  <constraintErrorMessage>Operation mode must be dr, tun, or nat</constraintErrorMessage>
+                </properties>
+                <defaultValue>dr</defaultValue>
+              </leafNode>
+              <leafNode name="algorithm">
```

Comment:
Please use an XML include as this is defined multiple times

Reply:
This option provides a load balancing algorithm. I can't find another existing load balancing cli to use the same option. If so, please let me know!

Reply:
It is in this very same PR.

Reply:
Format in this cli definition, there is only one `mode` and `algorithm` option. Do you mean that the `disable` and `ip` fields can be extracted?

Reply:
My fault, wrong search. 

But all XML CLI duplicates must be avoided and be implemented as re-usable include files. Please switch to the generic disable building block.